script/Ciel/Day 1/01_01_CIEL01_3_3.txt

```diff
@@ -181,12 +187,14 @@ If there's anything you need, please let me know. I'm usually here after school.
 [sha:3969fe8242b5083a5da3fc43c8b6fafa2edb9535]{
 -- Page 202, Offset 969.
 -- それは公私混同というか、秘密基地というか。
-Is this mixing personal and public affairs or more like a secret base, I wonder?
+Does that count as mixing personal or public affairs?
+ Or is she using it more as a secret base?
 }
 [sha:930c50a831aa39178102a33f7f7beefc1f2fc471]{
 -- Page 202, Offset 970.
 -- シエル先輩に誘われるのは光栄だけど、そんなんで茶道部復興はなるのだろうか？
-It would be an honor to be invited by Ciel-senpai to join the tea ceremony club, but will that be enough to revive it?
+I'm honored that Ciel-senpai has given me a standing invitation to visit,
+ but will that really help her revive the tea ceremony club?
 }
 [sha:7b734f2140370ed85bae49409938d4317fa1027a]{
 -- Page 203, Offset 971.
```

Comment:
Predator makes Shiki sound like a sex offender. Which is true, I guess, but not what is meant. Maybe alpha?
>But you sure are nosy, aren't you? Sure, I did invite you, but I didn't expect you to come all the way to the tea ceremony room. Perhaps you're more alpha-male than you look?